assets/js/modules/analytics-4/components/dashboard/EnhancedMeasurementActivationBanner/SetupBanner.js

```diff
@@ -44,22 +44,28 @@ import {
 	MODULES_ANALYTICS_4,
 } from '../../../datastore/constants';
 import { ERROR_CODE_MISSING_REQUIRED_SCOPE } from '../../../../../util/errors';
-import { getTimeInSeconds } from '../../../../../util';
+import { DAY_IN_SECONDS, getTimeInSeconds } from '../../../../../util';
 import { SpinnerButton } from 'googlesitekit-components';
 import BannerNotification from '../../../../../components/notifications/BannerNotification';
 import SuccessGreenSVG from '../../../../../../svg/graphics/ga4-success-green.svg';
 import ErrorNotice from '../../../../../components/ErrorNotice';
+import SurveyViewTrigger from '../../../../../components/surveys/SurveyViewTrigger';
 
 const { useDispatch, useSelect } = Data;
 
 export default function SetupBanner( {
+	isDismissed,
 	onSubmitSuccess,
 	onDismiss,
 	children,
 } ) {
 	const [ errorNotice, setErrorNotice ] = useState( null );
 	const [ isSaving, setIsSaving ] = useState( false );
 
+	const usingProxy = useSelect( ( select ) =>
+		select( CORE_SITE ).isUsingProxy()
+	);
```

Comment:
This is also not needed because we already check for this in the `SurveyViewTrigger` component 😄.

Reply:
Right, whoops! 😅